### PR TITLE
Test tutorial 2

### PR DESCRIPTION
**Pull request by sujeet14788**
Hello JP,
Please review the changes for tutorial 02 and provide your feedback.

### CODE REVIEW

File: ModelDerivative_02/instructions/task-2.md [hunk 1]
```diff
@@ -19,46 +19,80 @@ In this tutorial, you will use a Postman environment variable named `ossBucketKe
...
+   ![Minutes expiration](../images/task2_minutes_expiration_tt2.png "Minutes expiration")
 
-   3. Click the **Environment quick look** icon to hide the variables.
+4. Click **Send**. A script in the **Tests** tab updates the following Postman environment variables:
 
-2. In the Postman sidebar, click **Task 2 - Upload Source File to OSS > PUT Upload Zip File to OSS**. The request loads.
+   | Variable Name              | Description                                                                                 |
+   |----------------------------|---------------------------------------------------------------------------------------------|
+   | UploadKey | The upload key to upload the file.                                                                           |
+   | ContentUploadSignedURL | URN to upload source file                                                                       |
```
Comment (JayanathP) [inline ModelDerivative_02/instructions/task-2.md]: ContentUploadSignedURL - The signed upload URL you must use to upload the zip file.

File: ModelDerivative_02/instructions/task-2.md [hunk 2]
```diff
@@ -19,46 +19,80 @@ In this tutorial, you will use a Postman environment variable named `ossBucketKe
...
-5. Click **Send**. If the request is successful, you should see a screen similar to the following image.
+4. Click **Send**. If the request is successful, you should see a screen similar to the following image.
 
-    ![Successful Bucket Creation](../images/task2-sucessfull_bucket_creation.png "Successful Bucket Creation")
+   ![Successful Bucket Creation](../images/task2_creat_bucket_tt2.png "Successful Bucket Creation")
+    
+## Obtain Signed URL
 
-## Upload zip file to OSS
+1. In the Postman sidebar, click **Task 2 - Upload Source File to OSS > GET Obtain Signed URL**. The request loads.
```
Comment (JayanathP) [inline ModelDerivative_02/instructions/task-2.md:34]: Might be good to add a statement here informing users why they must follow this step.

Before you upload a file to OSS, you must obtain a signed upload URL for the file. To obtain a signed upload URL:

File: ModelDerivative_02/instructions/task-2.md [hunk 3]
```diff
@@ -19,46 +19,80 @@ In this tutorial, you will use a Postman environment variable named `ossBucketKe
...
+   |----------------------------|---------------------------------------------------------------------------------------------|
+   | UploadKey | The upload key to upload the file.                                                                           |
+   | ContentUploadSignedURL | URN to upload source file                                                                       |
+   
+You should see a screen similar to the following image,
+    
+   ![Signed url](../images/task2_obtain_signed_url_tt2.png "Signed url")
+   
+## Upload the file
+
```
Comment (JayanathP) [inline ModelDerivative_02/instructions/task-2.md:58]: Might be good to add an introductory statement here. For example,

Now that you have obtained a signed upload URL, you can go ahead and upload the zip file to OSS.

File: ModelDerivative_02/instructions/task-2.md [hunk 4]
```diff
@@ -19,46 +19,80 @@ In this tutorial, you will use a Postman environment variable named `ossBucketKe
...
 
 4. Click **Select File** and select the file *Tuner.zip*, which you downloaded in step 1.
 
-    ![Select file button](../images/task2-select_files_button.png "Select file button")
+   ![Select file button](../images/task2_upload_file_tt2.png "Select file button")
+   
+6. Click **Send** to upload the file.
+
+## Finalize Upload
+
```
Comment (JayanathP) [inline ModelDerivative_02/instructions/task-2.md:76]: Might be good to add an introductory paragraph for Finalize Upload. For example:

Although you uploaded the source file in one go, it is possible to split a file into chunks and upload the file one chunk at a time. Once all the chunks are uploaded you must inform OSS that the upload operation is complete. Even though you uploaded the file in one go, you must finalize the upload by informing OSS that the upload is done. To finalize the upload:

**Review by JayanathP — CHANGES_REQUESTED** [review]

File: ModelDerivative_02/instructions/task-2.md (hunk 1 above)
Comment (sujeet14788) [reply ModelDerivative_02/instructions/task-2.md]: Corrected

File: ModelDerivative_02/instructions/task-2.md (hunk 2 above)
Comment (sujeet14788) [reply ModelDerivative_02/instructions/task-2.md:34]: Corrected

File: ModelDerivative_02/instructions/task-2.md (hunk 3 above)
Comment (sujeet14788) [reply ModelDerivative_02/instructions/task-2.md:58]: Done

File: ModelDerivative_02/instructions/task-2.md (hunk 4 above)
Comment (sujeet14788) [reply ModelDerivative_02/instructions/task-2.md:76]: Done

**Comment by sujeet14788** [comment]
Implemented the feedback.

**Pull request merged**